internal/controller/warehouses/warehouses.go

```diff
@@ -238,6 +243,7 @@ func (r *reconciler) syncWarehouse(
 ) (kargoapi.WarehouseStatus, error) {
 	status := *warehouse.Status.DeepCopy()
 	status.ObservedGeneration = warehouse.Generation
+	status.LastFreight = warehouse.Status.LastFreight
```

Comment:
I'm not sure I understand the reason for this line. Isn't a copy of `LastFreight` included in the deep copy that was already performed?

Reply:
removed:
https://github.com/akuity/kargo/pull/1477/commits/953eed13f54be0cc23c93edcf84962335162850b